cmake/CMakeLists.txt

```diff
@@ -374,133 +386,133 @@ if(BUILD_IFCGEOM)
 
     # Open CASCADE
     if(WITH_OPENCASCADE)
-    if("${OCC_INCLUDE_DIR}" STREQUAL "")
```

Comment:
Since OCCT now support to export their cmake config files, we should use `find_package` instead of the manually find functions.
See https://github.com/Open-Cascade-SAS/OCCT/blob/22d437b771eb322dcceec3ad0efec6876721b8a9/CMakeLists.txt#L1357